### ISSUE TITLE
Binary Classification examples did not mention about threshold value when using Softmax

### ISSUE THREAD

**Issue by KianYang-Lee** [CertifaiAI/cdle-traininglabs#155]
**Describe the Enhancement Suggestions**
For binary classification example, threshold is a hyperparameter to be set to produce different model evaluation metrics. Examples need to be given on how to set threshold for it, else the performance of using softmax and sigmoid would be different.

**Comment by KianYang-Lee** [comment]
Refer to this for examples on using Evaluation to modify threshold https://deeplearning4j.org/api/latest/org/nd4j/evaluation/classification/Evaluation.html

**Comment by KianYang-Lee** [comment]
Closing due to inactivity
